constraints.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+// Module
+#define DEFAULT_SOCKET_PATH "/tmp/viam.csi.sock"
+#define RESOURCE_TYPE "CSICamera"
+#define API_NAMESPACE "viam"
+#define API_TYPE "camera"
+#define API_SUBTYPE "csi"
+
+// GST
+#define GST_GET_STATE_TIMEOUT 1 
+#define GST_CHANGE_STATE_TIMEOUT 5
+
+// Camera
+#define FAKE_CAMERA false
```

Comment:
How to do we set this to `true` to use the `FAKE_CAMERA` functionality?

Reply:
Great point - added `fake` attribute to config so you can set in app.

Reply:
If it's for internal testing purposes we probably don't want to document that and confuse users. Or see my other comment about leaving it out of this PR.

Reply:
Removed fake camera.